tests/pfcwd/test_pfcwd_function.py

```diff
@@ -208,6 +244,24 @@ def setup_port_params(self, port, init=False):
              self.pfc_wd['test_port_ids'] = self.pfc_wd['test_port_id']
          self.queue_oid = PfcCmd.get_queue_oid(self.dut, port, self.pfc_wd['queue_index'])
 
+    def update_queue(self, port):
+         if self.pfc_wd['queue_index'] == 4:
+            self.pfc_wd['queue_index'] = self.pfc_wd['queue_index'] - 1
+         else:
+            self.pfc_wd['queue_index'] = self.pfc_wd['queue_index'] + 1
+         logger.info("Current queue: {}".format(self.pfc_wd['queue_index']))
+         self.queue_oid = PfcCmd.get_queue_oid(self.dut, port, self.pfc_wd['queue_index'])
+
+    def setup_mmu_params(self, port):
+         self.pg_profile, self.alpha = PfcCmd.get_mmu_params(self.dut, port)
+
+    def update_mmu_params(self, mmu_action):
+        if mmu_action == "change":
+            PfcCmd.update_alpha(self.dut, self.pg_profile, int(self.alpha) + 1)
```

Comment:
Do you have a range in mind? I can accordingly decrease/increase alpha if it is above/below that range and increase if it is within the range.  Since we are changing only for lossless profiles, the alpha value isn't too big currently

Reply:
[-6, 3] should be a reasonable range.